src/Result.mo

```diff
@@ -44,7 +44,7 @@ public func chain<R1, R2, Error>(
   x : Result<R1, Error>,
   y : R1 -> Result<R2, Error>
 ) : Result<R2, Error> {
-  switch x {
+  switch(x) {
```

Comment:
Our style guide adopts the usual C/Java/etc style and wants a space here:
```suggestion
  switch (x) {
```
But it also allows omitting the parens.